go/client/ui.go

```diff
@@ -917,6 +917,14 @@ func (ui *UI) PromptYesNo(_ libkb.PromptDescriptor, p string, def libkb.PromptDe
 	return ui.Terminal.PromptYesNo(p, def)
 }
 
+func (ui *UI) GetSize() (width int, height int, error error) {
```

Comment:
Do we want the `error` here? It doesn't seem to be used. in `Terminal.GetSize()` it just defaults to 80,24 if `open()` errors